reference/5.1/Microsoft.PowerShell.Utility/Update-TypeData.md

```diff
@@ -3,131 +3,180 @@ external help file: Microsoft.PowerShell.Commands.Utility.dll-Help.xml
-For more information about the In operator, see about_Comparison_Operators (https://go.microsoft.com/fwlink/?LinkID=113217).
+The value of the **Value** property is a script that calculates the current annual quarter. The
+script block uses the `$this` automatic variable to represent the current instance of the object and
+the In operator to determine whether the month value appears in each integer array. For more
+information about the `-in` operator, see
+[about_Comparison_Operators](../Microsoft.PowerShell.Core/about/about_Comparison_Operators.md).
 
 The second command gets the new Quarter property of the current date.
 
 ### Example 4: Update a type that displays in lists by default
+
+This example shows how to set the properties of a type that displays in lists by default, that is,
+when no properties are specified. Because the type data is not specified in a `Types.ps1xml` file, it
+is effective only in the current session.
+
+```powershell
+Update-TypeData -TypeName "System.DateTime" -DefaultDisplayPropertySet "DateTime, DayOfYear, Quarter"
+Get-Date | Format-List
 ```
-PS C:\> Update-TypeData -TypeName "System.DateTime" -DefaultDisplayPropertySet "DateTime, DayOfYear, Quarter"
-PS C:\> Get-Date | Format-List
+
+```Output
 Thursday, March 15, 2012 12:00:00 AM
 DayOfYear : 75
 Quarter   : Q1
 ```
 
-This example shows how to set the properties of a type that displays in lists by default, that is, when no properties are specified.
-Because the type data is not specified in a Types.ps1xml file, it is effective only in the current session.
-
-The first command uses the **Update-TypeData** cmdlet to set the default list properties for the **System.DateTime** type.
-The command uses the *TypeName* parameter to specify the type and the *DefaultDisplayPropertySet* parameter to specify the default properties for a list.
-The selected properties include the new Quarter script property that was added in a previous example.
+The first command uses the `Update-TypeData` cmdlet to set the default list properties for the
+**System.DateTime** type. The command uses the **TypeName** parameter to specify the type and the
+**DefaultDisplayPropertySet** parameter to specify the default properties for a list. The selected
+properties include the new **Quarter** script property that was added in a previous example.
 
-The second command uses the Get-Date cmdlet to get a **System.DateTime** object that represents the current date.
-The command uses a pipeline operator (|) to send the **DateTime** object to the Format-List cmdlet.
-Because the **Format-List** command does not specify the properties to display in the list, Windows PowerShell uses the default values that were established by the **Update-TypeData** command.
+The second command uses the `Get-Date` cmdlet to get a **System.DateTime** object that represents
+the current date. The command uses a pipeline operator (|) to send the **DateTime** object to the
```

Comment:
The pipeline should probably be codefenced.

Reply:
Fixed